editor/import/editor_import_collada.cpp

```diff
@@ -1755,7 +1755,7 @@ void EditorSceneFormatImporterCollada::get_extensions(List<String> *r_extensions
 	r_extensions->push_back("dae");
 }
 
-Node *EditorSceneFormatImporterCollada::import_scene(const String &p_path, uint32_t p_flags, int p_bake_fps, List<String> *r_missing_deps, Error *r_err) {
+Node *EditorSceneFormatImporterCollada::import_scene(const String &p_path, uint32_t p_flags, const Dictionary &p_options, int p_bake_fps, List<String> *r_missing_deps, Error *r_err) {
```

Comment:
I would probably still pass the regular options map here, I suggest using the dictionary only for the function you expose for binding to script via GDVIRTUAL.

Reply:
I am having trouble with this. Can you assist?

Reply:
Yes, I Suggest just pass `const Map<StringName, Variant> &p_options` in the C++ side, in the bindings side do:
```C++
	GDVIRTUAL4R(Object *, _import_scene, String, Dictionary, uint32_t, uint32_t)
	GDVIRTUAL4R(Ref<Animation>, _import_animation, Dictionary, String, uint32_t, uint32_t)
```

and pass a conversion of the options to a dictionary. You can make a small helper static function to use in both of those when doing the GDVIRTUAL_CALL.

Reply:
Updated.